### PR TITLE
Initial docs for Bellissima Routes

### PR DESCRIPTION
**Pull request by enkelmedia**
## Description

Added initial docs (needs review) for routes, I'm sure that there are more to this but figured that this is good start.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
Umbraco v14 new backoffice

### CODE REVIEW

**Review by alina-tincas — CHANGES_REQUESTED** [review]
Hi @enkelmedia thank you for the PR, much appreciated 🙌 

I have few suggestions, can you please take a look into them? 😊

**Comment by enkelmedia** [comment]
Great feedback @alina-tincas! 

Thanks for the fixes! 

Let me know if you have other things that you would like to change =D

File: 14/umbraco-cms/extending-backoffice/routes.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    },
+  },
+  {
+    path: '',
+    redirectTo: 'people',
+  },
+];
+```
+{% hint style="info" %}
+The order in which the routes are defined is important as the first match will be used so make sure to add more specific routes in the beginning.
```
Comment (alina-tincas) [inline 14/umbraco-cms/extending-backoffice/routes.md:52]: fixinging lint error
```suggestion
The order in which the routes are defined is important as the first match will be used. So make sure to add more specific routes in the beginning.
```

File: 14/umbraco-cms/extending-backoffice/routes.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* **Workspace**: The [Workspace](extension-types/workspaces/README.md) concept has built-in features to facilitate editing of an entity of a certain entity type. It is used by many entities in the backoffice like content, media, content types, data types, dictionaries and so on.
+* **Custom element**: A Custom Element is a section that can be configured to use any web component as the **entry point**. The `element()` can be configured in the manifest. By doing this we'll disable the possibility of using dashboards and section views for the section since they will not be automatically routed/rendered. This option should be used only when necessary.
+
+## Building routing
+Almost any component can host routable sub-components by defining a list of routes and render a `umb-router-slot` element. Let's assume we have a custom section with pathname `custom-section` and a `sectionView` with pathname `organization`. In this context we can create an element with routes, like this:
```
Comment (alina-tincas) [inline 14/umbraco-cms/extending-backoffice/routes.md:22]:
```suggestion
Almost any component can host routable sub-components by defining a list of routes and render a `umb-router-slot` element. Let's assume we have a **custom section** with pathname `custom-section` and a **sectionView** with pathname `organization`. In this context we can create an element with routes, like this:
```

**Review by alina-tincas — APPROVED** [review]

**Comment by alina-tincas** [comment]
I have nothing else to add for the moment, so I will merge this in 🙌

**Comment by enkelmedia** [comment]
Great!

Maybe if @nielslyngsoe or someone from the team want to check that I’m not straight out lying about something 😂

**Comment by alina-tincas** [comment]
I believe you are not lying 😆 I think the content makes sense hence I have approved and merged it. The team is quite busy until release at end of may so they will not be able to check this until then. It is however in our plan for the frontend team to check the new backoffice articles from extending-backoffice around final release or at least shortly after the release depending how busy they are. 👍

**Pull request merged**